CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 48.2.0 (September 04, 2020)
```

Comment:
Note: The `version.rb` file also needs to be updated, however the date above, and the change for the version.rb file should be in a separate commit (with no other changes).

Reply:
Done